priv/leo_gateway.conf

```diff
@@ -79,6 +79,9 @@ http.max_keepalive = 4096
 ## HTTP timeout for reading body
 ## http.timeout_for_body = 15000
 
+## HTTP sending chunk length
+## http.sending_chunk_len = 131072
+
```

Comment:
The default should NOT be affected to existing user environments.

s/131072/5242880/